app/presenters/hyrax/users_presenter.rb

```diff
@@ -0,0 +1,55 @@
+module Hyrax
+  class UsersPresenter
+    attr_reader :query, :authentication_key
+
+    def initialize(query:, authentication_key:)
+      @query = query
+      @authentication_key = authentication_key
+    end
+
+    # @return [Array] an array of Users
+    def users
+      @users = search(query)
+    end
+
+    def user_count
+      ::User.not_batch_or_audit_user.count
+    end
+
+    def repository_admin_count
+      count_admin = 0
+      ::User.not_batch_or_audit_user.each do |user|
+        count_admin += 1 if admin?(user)
+      end
+      count_admin
+    end
+
+    def user_roles(user)
+      roles = user.groups
+      return roles if roles.any?
+      ['']
```

Comment:
As this is a presenter, if roles come back as nil, I had to qualify, it misaligns the columns on the view. The logic needed to either be in here or in the view file so I put it here.